jalien.sh

```diff
@@ -0,0 +1,37 @@
+package: JAlien
+version: "%(tag_basename)s"
+tag: master
+source: https://gitlab.cern.ch/jalien/jalien.git
+requires:
+ - Java
+ - XRootD
+build_requires:
+ - Java
+valid_defaults:
+ - jalien
+---
+#!/bin/bash -e
+
+rsync -av $SOURCEDIR/ ./
+./compile.sh users
+mkdir -p $INSTALLROOT/lib
+cp alien-users.jar $INSTALLROOT/lib/
```

Comment:
@costing do we still need this distinction between "user" installation and all the rest? Is there a relevant difference in size? Having a single installation would make deployment easier.

@brinick, @costing: how is JAliEn launched? `java -jar alien-users.jar`? If this is the case, I suggest to:

* Keep the `.jar` inside `lib` and the `CLASSPATH` as they are
* Add a dummy `jalien` binary in `bin` (export `PATH` accordingly too) that does:

```bash
#!/bin/bash -e
exec java -jar alien-users.jar "$@"
```

so that one can run it by simply typing `jalien`.

Reply:
alien-users.jar is the only needed file. The command line will be composed by whichever service is running, picking one or other entry point in it. Yes, we might have this helper script but not sure it helps much, especially is PATH and CLASSPATH are correctly exposed.

Reply:
Say, I prefer if we ask our users to run:

```bash
jalien ...
```

rather than:

```bash
java -jar alien-users.jar
```

As a matter of fact: I'd really prefer to run `jalien` like this myself, I find the full command rather obnoxious 🙂

For the record, adding this script is just a matter of doing in the recipe (no changes to JAliEn):

```bash
mkdir $INSTALLROOT/bin
cat > $INSTALLROOT/bin/jalien <<EOF
#!/bin/bash -e
exec java -jar alien-users.jar "$@"
EOF
chmod 0755 $INSTALLROOT/bin/jalien
```

so I really believe we should do it.

Reply:
As discussed yesterday with @costing and @brinick, we drop the creation of the wrapper script and we `rsync` the `bin` directory of the JAliEn repository to `$INSTALLROOT/bin`.

Reply:
Ok, the "bin/jalien" script is there and a "0.0.0" tag too, to switch to using tags instead of master from now on.